samples/net/sockets/http_server/src/static_web_resources/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
```

Comment:
proposing a couple changes to make sure it's valid HTML (this one is mostly a warning but the changes to input tags are fixing actual invalid HTML)

```suggestion
<html lang="en">
```